Source/FolioReaderConfig.swift

```diff
@@ -37,12 +39,54 @@ public enum FolioReaderScrollDirection: Int {
     }
 }
 
+// MARK: - ClassBasedOnClickListener
+
+/**
+A `ClassBasedOnClickListener` takes a closure which is performed if a given html `class` is clicked. The closure will reveice the content of the specified parameter.
+
+Eg. A ClassBasedOnClickListener with the className "quote" and parameterName "id" with the given epub html content "<section class="quote" id="12345">" would call the given closure on a click on this section with the String "12345" as parameter.
+
+*/
+public struct ClassBasedOnClickListener {
+
+	/// The name of the URL scheme which should be used. Note: Make sure that the given `String` is a valid as scheme name.
+	public var schemeName			: String
+
+	/// The HTML class name to which the listener should be added.
+	public var className			: String
+
+	/// The name of the parameter whose content should be passed to the `onClickAction` action
+	public var parameterName		: String
+
+	/// The closure which will be called if the specified class was clicked.
+	public var onClickAction		: ((parameterContent: String?) -> Void)
+
+	/// Initializes a `ClassBasedOnClickListener` instance. Append it to the `classBasedOnClickListeners` property from the `FolioReaderConfig` to receive on click events.
+	public init(schemeName: String, className: String, parameterName: String, onClickAction: ((parameterContent: String?) -> Void)) {
```

Comment:
In addition we would change the `className` to `querySelector` and maybe add a new `Bool` attribute `selectAll` with `true` by default, so I can choose to select only first element or all elements. What do you think?